packages/tasks/src/model-libraries.ts

```diff
@@ -680,6 +680,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"safetensors" OR path_extension:"pt"`,
 	},
+	"mochi-1-preview": {
+		prettyLabel: "Mochi Preview",
+		repoName: "mochi-1-preview",
```

Comment:
Depending on what's decided above, I would suggest to make this naming more consistent. For instance

```suggestion
		prettyLabel: "Mochi 1",
		repoName: "Mochi 1",
```